pygments/lexers/tcl.py

```diff
@@ -77,6 +77,8 @@ def _gen_command_rules(keyword_cmds_re, builtin_cmds_re, context=""):
             (r'\(', Keyword, 'paren'),
             (r'\[', Keyword, 'bracket'),
             (r'\{', Keyword, 'brace'),
+            (r'\\(.)', String.Escape),
+            (r'\$\s', Text),
```

Comment:
`$` is a character literal but `$x` is a variable.

Reply:
This instant is still correctly lexed by the lexer because the change only considers the case where `$` is a standalone character.